data.sql

```diff
@@ -8,3 +8,19 @@ INSERT INTO animals (name, date_of_birth, escape_attempts,neutered,weight_kg) VA
 
```

Comment:
- [ ] In your Readme, you have broken links to the MIT license. Please make sure you have a working link.